packages/matchbox/src/components/ThemeProvider/globalStyles.js

```diff
@@ -112,6 +112,7 @@ export default `
   a,
   a:visited {
     color: ${tokens.color_blue_700};
+    text-decoration: underline;
```

Comment:
Thanks for doing this - we'll end up removing the underline on a case-by-case basis in the app, but having this as the default is the right thing to do.